static/app/views/issueList/actions/index.tsx

```diff
@@ -273,10 +273,15 @@ function useSelectedGroupsState() {
   const selectedIds = useLegacyStore(SelectedGroupStore);
 
   const selected = SelectedGroupStore.getSelectedIds();
-  const projects = [...selected]
-    .map(id => GroupStore.get(id))
-    .filter((group): group is Group => !!(group && group.project))
-    .map(group => group.project.slug);
+  const projects: string[] = [];
+
+  for (const selectedGroup of selected) {
+    const group = GroupStore.get(selectedGroup);
+
+    if (group && group.project) {
+      projects.push(group.project.slug);
+    }
+  }
```

Comment:
i feel like the map and filter should just be written better

maybe
```ts
const projects = selected
    .map(id => GroupStore.get(id)?.project.slug)
    .filter(defined)
```

Reply:
It does not work because selected is a Set :/